### PR TITLE
Create TRANSLATING.md file with instructions

### PR DESCRIPTION
**Pull request by mribeirodantas**
Signed-off-by: Marcel Ribeiro-Dantas <mribeirodantas@seqera.io>

### CODE REVIEW

File: TRANSLATING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# 🌐 <abbr title="translation">Translation</abbr> Guide
```
Comment (ewels) [inline TRANSLATING.md:1]: Why an abbreviation?

File: TRANSLATING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Table of contents:
```
Comment (ewels) [inline TRANSLATING.md:3]: It should autogenerate this in the sidebar, no? Not sure we need it manually as well?

File: TRANSLATING.md (hunk 1 above)
Comment (mribeirodantas) [reply TRANSLATING.md:1]: I copy-pasted this from another repo that had a document for translating. Now that I look at it, there's no reason 😓 I will remove it.

File: TRANSLATING.md (hunk 2 above)
Comment (mribeirodantas) [reply TRANSLATING.md:3]: You're right. I forgot of this GitHub feature 😬

**Pull request merged**